mkosi/config.py

```diff
@@ -996,6 +997,14 @@ class MkosiConfigParser:
             paths=("mkosi.repart",),
             help="Directory containing systemd-repart partition definitions",
         ),
+        MkosiConfigSetting(
+            dest="sector_size",
+            long="--sector-size",
+            name="SectorSize",
```

Comment:
This can be dropped, it's inferred automatically

Reply:
Just drop long and name or all 3?

Reply:
Drop long and name